tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/AppFilterSuite.scala

```diff
@@ -528,4 +528,146 @@ class AppFilterSuite extends FunSuite {
       }
     }
   }
+
+  case class TestConjunctionAndDisjunction(
+      fileName: String, fsTime: Long, appName: String,
```

Comment:
not really testing fsTime and by code we aren't applying any/all to it properly

Reply:
I will add tests for fsTime along with appFilters with `any/all`